spec/feature/confirm_email_prompt_spec.rb

```diff
@@ -54,7 +54,7 @@
   end
 
   def given_i_have_logged_in
-    visit user_session_path
```

Comment:
This wasn't root but was another login visit. I thought it makes sense for this to be consistent?